README.md

```diff
@@ -1,2 +1,21 @@
-# template-repo
-Various materials related to or pertaining to the branding of OpenZiti
+# builder container for ziti-edge-tunnel
+
+This container image builds the `ziti-edge-tunnel` binary for Linux from source with statically-linked crypto libraries. This container image is available from Docker Hub as [openziti/ziti-tunnel-sdk-c-builder](https://hub.docker.com/r/openziti/ziti-tunnel-sdk-c-builder).
+
+## Build ziti-edge-tunnel
+
+To build the ziti-edge-tunnel binary, run the following command in the root of the **ziti-tunnel-sdk-c** repository:
+
+```bash
+docker run --rm --volume "${PWD}:/github/workspace" openziti/ziti-tunnel-sdk-c-builder
```

Comment:
this seems weird it I run the docker locally

Reply:
Do you mean you got an unexpected result?